apps/core-app/src/main/modules/box-tool/addon/apps/app-provider.test.ts

```
@@ -644,6 +655,70 @@ describe('appProvider rebuild maintenance', () => {
+              if (values.displayName) {
+                updatedDisplayName = values.displayName
+              }
+            })
+          }))
+        }))
+      }),
+      addFileExtensions: vi.fn(async () => undefined)
+    }
+    privateProvider.searchIndex = { indexItems: vi.fn(async () => undefined) }
+    privateProvider.fetchExtensionsForFiles = vi.fn(async (files: unknown[]) =>
+      files.map((file) => ({
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)